src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -715,6 +715,10 @@ const getIconColor = computed(() => props.theme === 'light' ? KUI_COLOR_TEXT_NEU
 </script>
 
 <style lang="scss" scoped>
+/* Component variables */
+
+$kCodeBlockDarkLineMatchBackgroundColor: rgba(255, 255, 255, 0.12); // we don't have a token for this
```

Comment:
"dark line match background color" - I have no idea what this means, or what it's used for 😬  Could you add a comment or a better name for the var?

Reply:
Done